server/index.js

```diff
@@ -26,8 +24,7 @@ app.get('/reviews', (req, res) => {
 });
 
 app.get('/reviews/item', (req, res) => {
-  console.log(req.originalUrl);
-  let courseId = req.query.id;
+  let courseId = req.query.courseId;
   let reviews;
```

Comment:
Where are these being defined?

Reply:
I'm not sure what you mean here...?

Reply:
What is being assigned to these variables?